### PR TITLE
setup basic docs pages for pangeo-stacs

### PR DESCRIPTION
**Pull request by jhamman**
@scottyhq suggested we put up docs page that includes some user facing instructions for how to use the images here and a place where we can document what packages (and versions) are included in each image. I've taken the liberty to reuse most of @rabernat's pangeo-datastore setup.  

The images page renders to something like this: 

image

I'll be keen to hear from @scottyhq (and others) what else would be useful here.

### CODE REVIEW

File: docs/build_stacks_rst.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        #     'docker', 'run', '-it',
+        #     '--entrypoint=""',
+        #     image, 
+        #     'conda', 'list']
+        # joined_args = " ".join(args)
+        # print('running ', joined_args)
+        # out = subprocess.check_output(joined_args, shell=True)
+        # out = out.decode().replace('#', '') + '\n'
+        # print(out, type(out))
+        # return out
```
Comment (jhamman) [inline docs/build_stacks_rst.py]: This isn't working yet. We need to figure out the best way to clean up the output from `conda-list` so it can be rendered in rst/html.

File: docs/build_stacks_rst.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        d.newline()
+
+        for image in self.images:
+            d.h3(image)
+            d.newline()
+
+            conda_list = self.conda_list(image)
+
+            metadata = {'url': f'https://hub.docker.com/r/pangeo/{image}',
+                        'onbuild-url': f'https://hub.docker.com/r/pangeo/{image}-onbuild'}
```
Comment (jhamman) [inline docs/build_stacks_rst.py]: This metadata dict is a place holder. @scottyhq - what do you want to see here. The microbader fields give a decent sampling of the available metadata: https://microbadger.com/images/pangeo/pangeo-notebook

**Comment by jhamman** [comment]
I'm going to merge this now so I can debug any final deployment issues. Feel free to add comments here and I can follow up with a fine tuning PR.

**Comment by jhamman** [comment]
Now live here: https://pangeo-data.github.io/pangeo-stacks/images

File: docs/build_stacks_rst.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+            mb_meta = requests.get(f'https://api.microbadger.com/v1/images/{image}').json()['Versions'][0]  # latest version
```
Comment (jhamman) [inline docs/build_stacks_rst.py]: using microbadger was a nice idea but after seeing how long it takes for them to update their API, we should probably just go with `docker image inspect $image`. This also returns json and shouldn't be too hard to parse.

**Comment by scottyhq** [comment]
this is fantastic. thanks for realizing the vision so quickly @jhamman !
